plugins/out_es/es.c

```diff
@@ -852,6 +850,8 @@ static void cb_es_flush(struct flb_event_chunk *event_chunk,
     }
 #endif
 
+    flb_http_add_header(c, "Content-Type", 12, "application/x-ndjson", 20);
+
```

Comment:
You moved this line down so that its not part of the signed headers? 

I think this is not a long term maintainable solution if that's the case... what if another maintainer goes and switches it back? 

May be we need something like an option to pass a list of headers to flb_signv4_do that should not be signed?

Reply:
Also where is the fix for Content-Length not being signed?

Reply:
Noted. This was actually a part of Zhonghui's commits. No change on my part. The new "unsigned_headers" parameter will omit what we want. Following up on whether Content-Type should be omitted.

Reply:
Need to move this back up